metrics/deploy/rbac.yaml

```diff
@@ -61,6 +61,14 @@ rules:
     - get
     - list
     - watch
+  - apiGroups:
```

Comment:
Please add these to https://github.com/red-hat-storage/ocs-operator/blob/main/rbac/exporter-role.yaml file also. And, regenerate the bundle (`make update-generated` `make gen-latest-csv`)

Reply:
Done.